packages/terra-framework-docs/src/terra-dev-site/doc/slider/Examples.2/DefaultSlider.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import Slider from 'terra-slider';
+
+const DefaultSlider = () => (
+  <Slider minimumValue={0} maximumValue={100} labelName="Label" defaultValue={50} />
```

Comment:
At least one of these examples should show how to use the callback and value props to manage state.

Reply:
Added an example.